### PR TITLE
chore(workflow): migrate release tagging to github actions

### PR DESCRIPTION
**Pull request by akhilerm**
**Why is this PR required? What issue does it fix?**:
The PR migrates the release tagging process from travis to github actions as per [this](https://github.com/openebs/openebs/issues/3352) issue.

**What this PR does?**:
- add downstream tagging to release
- remove travis file
- update build status badge

**Does this PR require any upgrade changes?**:

**If the changes in this PR are manually verified, list down the scenarios covered and commands you used for testing with logs:**

**Any additional information for your reviewer?**:
_Mention if this PR is part of any design or a continuation of previous PRs_
- `GR_TOKEN` secret needs to be added to the repo before merging
- travis app integration can be removed from the repository
- Dependent on #86 

**Checklist:**
- [ ] Fixes #<issue number>
- [x] PR Title follows the convention of  `<type>(<scope>): <subject>`
- [ ] Has the change log section been updated?
- [ ] Commit has unit tests
- [ ] Commit has integration tests
- [ ] (Optional) Are upgrade changes included in this PR? If not, mention the issue/PR to track:
- [ ] (Optional) If documentation changes are required, which issue on https://github.com/openebs/openebs-docs is used to track them:

### CODE REVIEW

**Review by shubham14bajpai — APPROVED** [review]
LGTM

File: buildscripts/install-test-dep.sh [hunk 1]
```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+
+# Copyright 2020 The OpenEBS Authors. All rights reserved.
```
Comment (mynktl) [inline buildscripts/install-test-dep.sh]: should we change it to 2021?

Comment (akhilerm) [reply buildscripts/install-test-dep.sh]: Done.

**Comment by akhilerm** [comment]
@kmova Can you add the `GR_TOKEN` to this repository

**Review by mynktl — APPROVED** [review]

**Pull request merged**